kotlinx-coroutines-debug/build.gradle

```diff
@@ -38,6 +41,12 @@ if (rootProject.ext.jvm_ir_enabled) {
     }
 }
 
+java {
```

Comment:
Doesn't seem to be necessary for the latest `develop` where we have 1.8 target

Reply:
I attempted to remove this block, but then the build was failing. I managed to fix the build of this subproject by adding
```gradle
tasks.withType(JavaCompile) {
    targetCompatibility = JavaVersion.VERSION_1_8
}
```
However, then kotlinx-coroutines-test also required `targetCompatibility` set. At that point, I stopped. Do you think this should be pursued further?

Reply:
I think that's enough, thanks for checking it